pkgs/misc/crust/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, fetchurl
+, fetchFromGitHub
+, stdenv
+, pkgsCross
+, flex
+, bison
+, swig
+, buildPackages
+}:
+let
+  buildCrustFirmware = lib.makeOverridable (
+    { filesToInstall ? [ "build/scp/scp.bin" ]
+    , installDir ? "$out"
+    , defconfig
+    , extraConfig ? ""
+    , extraPatches ? [ ]
+    , extraMakeFlags ? [ ]
+    , extraMeta ? { }
+    , ...
+    } @ args:
+
+    stdenv.mkDerivation (rec {
```

Comment:
Nit: it would be better to use `finalAttrs:`, as in https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes